allennlp/training/metrics/bleu.py

```diff
@@ -120,6 +114,9 @@ def __call__(
 
         None
         """
+        # TODO: fix not being able to import this normally
```

Comment:
Any progress?

Reply:
Finally fixed this. It turns out that using `import x as y` works much more reliably than `from x import z`. Originally I tracked down where exactly the imports are circular and I thought maybe it can be resolved by moving some functions to a different module. But from what I can tell the circularity makes sense here so I just used:

`import allennlp.training.util as training_util`